_docs/technical/protocols/2017-01-13-binary.md

```diff
@@ -118,9 +118,11 @@ to describe maximum available value.
 
 ~~~ haskell
 newtype UnsignedVarInt a = UnsignedVarInt {getUnsignedVarInt :: a}
-    deriving (Eq, Ord, Show, Generic, NFData)
+    deriving (Eq, Ord, Show, Generic, NFData, Functor)
```

Comment:
Beginning of this file in the `Updated` comment should have new commit id after changes so everybody can see which commit was took as a baseline